ui/widgets/nuxeo-html-editor.js

```diff
@@ -134,6 +193,25 @@ import './quill/quill-snow.js';
         this._editor.getModule('toolbar').container.style.display = this.readOnly ? 'none' : '';
       }
     }
+
+    _onImageUpload() {
+      this.$.qlImage.click();
+    }
+
+    _onSearchImage() {
+      this.$.picker.open();
```

Comment:
would it make sense to return a promise here that resolves once the files are picked ? 🤔

Reply:
Could be a possibility but I find it a bit redundant considering that the element will fire an event once the selection is confirmed. It's true that there's also no way to followup if the user closes or cancels the dialog, but that also could be achieved with something like a `dismissed` event.